app/controllers/katello/api/v2/api_controller.rb

```diff
@@ -94,6 +97,12 @@ def scoped_search(query, default_sort_by, default_sort_order, resource = resourc
 
       query = query.order("#{sort_attr} #{params[:sort_order] || default_sort_order}")
 
+      if includes.is_a?(Proc)
```

Comment:
there isn't one, but i know examples where it would be needed once everything gets moved to scoped search.   Anything that has to go through more than a single relationship, such as these examples from the product show.json.rab and the repo's show.json.rabl:

product => provider => organization  
or
repo => product => sync_plan